[2023-03-13T19:25:17.681+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Spotify_Data_ETL.TransformData manual__2023-03-13T17:35:51.683738+00:00 [queued]>
[2023-03-13T19:25:17.686+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Spotify_Data_ETL.TransformData manual__2023-03-13T17:35:51.683738+00:00 [queued]>
[2023-03-13T19:25:17.687+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-13T19:25:17.687+0000] {taskinstance.py:1280} INFO - Starting attempt 14 of 15
[2023-03-13T19:25:17.687+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-13T19:25:17.695+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): TransformData> on 2023-03-13 17:35:51.683738+00:00
[2023-03-13T19:25:17.699+0000] {standard_task_runner.py:55} INFO - Started process 11217 to run task
[2023-03-13T19:25:17.702+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Spotify_Data_ETL', 'TransformData', 'manual__2023-03-13T17:35:51.683738+00:00', '--job-id', '271', '--raw', '--subdir', 'DAGS_FOLDER/spotifydag.py', '--cfg-path', '/tmp/tmp4jwbyhyh']
[2023-03-13T19:25:17.703+0000] {standard_task_runner.py:83} INFO - Job 271: Subtask TransformData
[2023-03-13T19:25:17.753+0000] {task_command.py:388} INFO - Running <TaskInstance: Spotify_Data_ETL.TransformData manual__2023-03-13T17:35:51.683738+00:00 [running]> on host 649b70e6a8b0
[2023-03-13T19:25:17.799+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ananta Moharana
AIRFLOW_CTX_DAG_ID=Spotify_Data_ETL
AIRFLOW_CTX_TASK_ID=TransformData
AIRFLOW_CTX_EXECUTION_DATE=2023-03-13T17:35:51.683738+00:00
AIRFLOW_CTX_TRY_NUMBER=14
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-13T17:35:51.683738+00:00
[2023-03-13T19:25:17.842+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pandas/util/_decorators.py:311: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  return func(*args, **kwargs)

[2023-03-13T19:25:17.856+0000] {logging_mixin.py:137} INFO - Index(['song_name', 'is_explicit', 'artist_name', 'artist_id'], dtype='object')
[2023-03-13T19:25:17.858+0000] {logging_mixin.py:137} INFO - False
[2023-03-13T19:25:17.858+0000] {logging_mixin.py:137} INFO - **************************
[2023-03-13T19:25:17.859+0000] {logging_mixin.py:137} INFO - Index(['song_name', 'track_id', 'is_explicit', 'artist_name', 'artist_id',
       'acousticness', 'danceability', 'duration_ms', 'energy',
       'instrumentalness', 'key', 'liveness', 'loudness', 'speechiness',
       'tempo', 'valence'],
      dtype='object')
[2023-03-13T19:25:17.861+0000] {logging_mixin.py:137} INFO - False
[2023-03-13T19:25:17.862+0000] {logging_mixin.py:137} INFO - **************************
[2023-03-13T19:25:17.862+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/transform.py", line 60, in transform_data
    print(song_arist_bridge.keys())
AttributeError: 'NoneType' object has no attribute 'keys'
[2023-03-13T19:25:17.872+0000] {taskinstance.py:1323} INFO - Marking task as UP_FOR_RETRY. dag_id=Spotify_Data_ETL, task_id=TransformData, execution_date=20230313T173551, start_date=20230313T192517, end_date=20230313T192517
[2023-03-13T19:25:17.880+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 271 for task TransformData ('NoneType' object has no attribute 'keys'; 11217)
[2023-03-13T19:25:17.914+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-13T19:25:17.928+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
